parallel-install/pkg/deployment/deployment.go

```diff
@@ -190,6 +196,11 @@ InstallLoop:
 			return err
 		}
 	}
+	// Only will be executed if Kyma deploy was successfull
```

Comment:
If I understand it right the post jobs would be executed after whole kyma deployment is performed ?

Reply:
The global post jobs will run after kyma is deployed which is right. May have a look [here](https://github.com/kyma-project/community/blob/main/internal/proposals/migration-hooks/migration-logic-diagram.png) to have a overview when jobs can be triggered.